public/resources/illustration_download.svg

```diff
@@ -0,0 +1,126 @@
+					<path id="Line" class="st5" d="M8.5,11.9c2.7,2.1,6.9,3.2,12.4,3.3c5.3,0.1,9.7-0.8,13.2-2.7"/>
+				</g>
+
+					<g id="illustration" transform="translate(73.367530, 63.335344) rotate(19.000000) translate(-73.367530, -63.335344) translate(25.867530, 13.335344)">
+					<path id="Combined-Shape" class="st6" d="M52.1,5.1L9.2,4l0,0c-1.3,0-2.5,0.7-3.1,1.9l0,0c-6.3,13.2-7.4,29.2-3.3,48
+						C7,73.1,13.3,87.1,21.5,96c0.4,0.4,0.8,0.7,1.4,0.9l0,0c3.1,1,9.5,1.8,19.3,2.3c10,0.5,26.3-0.4,48.9-2.8
+						c0.5-0.1,1-0.2,1.4-0.5c1.5-1,2-3,1-4.6l0,0C86.7,80.6,81.2,68.7,77,55.4c-3.5-10.9-3.5-22.5-0.2-34.8L52.1,5.1z"/>
+					<path id="Combined-Shape_1_" class="st1" d="M5.8,8.9C6,8.2,6.3,7.4,6.6,6.7l0,0c0.5-1.3,1.8-2.1,3.2-2.1l0,0L51.5,6l24.1,15.1
+						c-3.1,9.9-2.9,21.1,0.6,33.4C80.9,70.6,86.4,83,92.8,91.4l0,0c0.9,1.1,0.6,2.8-0.5,3.6c-0.4,0.3-0.8,0.5-1.3,0.5l0,0
+						c-0.5,0.1-1,0.1-1.5,0.1c0-0.1-0.1-0.1-0.1-0.2l0,0c-0.1-0.1-0.1-0.2-0.2-0.3c-5.3-9.3-11.3-20.2-15.7-33.5
+						c-2.1-6.3-4.7-17.2-4.4-26.7c0.3-9.5-3.4-7.8-11.3-6.3c-3.2,0.6-5.8-4.9-7.9-16.5c-0.3-1.6-1.6-2.7-3.2-2.7l0,0L5.8,8.9
+						L5.8,8.9z"/>
+
+						<g id="face" transform="translate(33.064221, 46.980973) rotate(5.000000) translate(-33.064221, -46.980973) translate(10.564221, 35.980973)">
+						<path id="Line_1_" class="st7" d="M5.5,8.6l0.7,3.2"/>
+						<path id="Line-Copy-3" class="st7" d="M39.5,7.5l0.7,3.2"/>
+						<path id="Path-5-Copy" class="st8" d="M43.9,3.6c-2.3-2.4-4.8-2.6-7.5-0.7"/>
+						<path id="Path-6" class="st8" d="M0.8,2.9c1.9-2.3,4.4-2.6,7.6-1"/>
+						<path id="Path-6_1_" class="st9" d="M33,16.7c-5,5.4-11.6,6-20,1.7"/>
+					</g>
+
+						<g id="Path" transform="translate(64.605652, 13.476265) rotate(-14.000000) translate(-64.605652, -13.476265) translate(54.105652, 2.976265)">
+						<path id="path-1" class="st10" d="M20.1,20L4.7,20c-2.2,0-4-1.8-4-4.1l0-15.9L20.1,20z"/>
+					</g>
+				</g>
+
+					<g id="Group-3_1_" transform="translate(124.343514, 68.259611) scale(-1, 1) rotate(22.000000) translate(-124.343514, -68.259611) translate(93.343514, 53.259611)">
+					<path id="Path-12_1_" class="st4" d="M61.9,11.7c-1.8-10.4-11-13.5-27.6-9.5C9.5,8.4,3.6-0.6,0.8,0.8s3.7,9.4,6.1,10.5
+						s-3.1-0.7-0.8,3.7c1.5,2.9,4.6,5.1,9.4,6.5c-1.9,1.8,1.2,3.9,9,6.2s13.7,2.3,17.6,0c-0.6,1.6,0.7,2.1,4,1.7
+						c3.3-0.5,6.2-1.6,8.8-3.4c2.6-1.7,4.4-3.8,5.5-6.3S62.1,14.6,61.9,11.7z"/>
+					<path id="Path-13_1_" class="st2" d="M60,13.3c-0.5-6.1-3.7-9.7-9.6-10.8c-4.1-0.8-8.9-0.2-22.5,2.7C9.1,9.2,0-2.9,3.3,5.2
+						s13.5,6.5,15.7,7s-4.4,0.8-6,0.8c-4.4,0-4.6-0.1-5.1,1s2.9,5.6,11.4,6.5c2.3,0.2,5.8,0.7,9.3,0.7c5.5,0,11.4-2.5,11.6-2.1
+						c0.2,0.4-6.3,2.5-11.6,3c-4,0.4-8.1-0.2-10,0c-4.3,0.4,4.7,5.3,13.6,5.9c8.9,0.6,10.3-2.4,11.5-2.6c1.2-0.2-2.6,2-0.7,2.6
+						c1.8,0.5,7.9-0.5,12.5-4.7C58.5,20.4,60,17.1,60,13.3z"/>
+					<path id="Line_2_" class="st5" d="M9.4,14.4c3.7,3.3,9,5,15.9,5.1c6.6,0.1,11.9-0.8,15.8-2.8"/>
+				</g>
+			</g>
+		</g>
+	</g>
+</g>
+</svg>
```

Comment:
I think we have an existing bug for this (Optimize images #63), but we could probably run all these through svgo or imagemin (at checkin or during build time) and shave a few KB.